web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -192,9 +174,15 @@ function addEvents(mangaList) {
         mangaList.filter(str);
     };
 
-    // On Register-but click, Trigger a new comic registration
-    var but = document.getElementById('register-but');
-    but.onclick = function() {
+    // On Register-but click, show the SidePanel in "REGISTER" mode
+    const regBtn = document.getElementById('register-but');
+    regBtn.onclick = function() {
+        showHideSidePanel(mangaList.MODE_REGISTER);
+    };
+
+    // On button-add click, Trigger a new comic registration
+    const addBut = document.querySelector('.button-add');
```

Comment:
Btw, we should be careful over `querySelector`. It's not taking into account DOM updates (or weirdly...).